### ISSUE TITLE
Setting lightSeparatorColor modify text shadow

### ISSUE THREAD

**Issue by ghost** [iGriever/TWSReleaseNotesView#6]
Adding the following configuration to example code :

```
[releaseNotesView setViewShadowColor:[UIColor redColor]];
[releaseNotesView setLightSeparatorColor:[UIColor greenColor]];
```

This leads to the following result where text have it's shadow color set to the lightSeparatorColor.

![130831-0001](https://f.cloud.github.com/assets/385357/1062658/1a79954c-1261-11e3-83a4-4ced423d34e9.png)

**Comment by iGriever** [comment]
Yes, correct. Will be fixed in 1.2.0.

**Comment by iGriever** [comment]
Fixed with version 1.2.0.

![ios simulator screen shot 01 set 2013 13 19 55](https://f.cloud.github.com/assets/2828592/1063880/0236fe78-12f9-11e3-9be7-26ceda79f325.png)

**Comment by iGriever** [comment]
Updated CocoaPods spec to version 1.2.0.
